com.unity.netcode.gameobjects/Runtime/Messaging/MessagingSystem.cs

```diff
@@ -55,6 +56,9 @@ public SendQueueItem(NetworkDelivery delivery, int writerSize, Allocator writerA
 
         private Dictionary<Type, uint> m_MessageTypes = new Dictionary<Type, uint>();
         private Dictionary<ulong, NativeList<SendQueueItem>> m_SendQueues = new Dictionary<ulong, NativeList<SendQueueItem>>();
+        private Dictionary<ulong, Dictionary<Type, int>> m_PerClientMessageVersions = new Dictionary<ulong, Dictionary<Type, int>>();
```

Comment:
Just making sure, this table is laid out like this: [ClientId][Messaage Type][Version] ?

Reply:
Correct.